maps/Sudan/Sudan_conflict_points

```diff
@@ -0,0 +1,85 @@
+// Author: Iman Smith
+// Description:
+//  Script for generating conflict event mask for Sudan from 2020-2024
+
+var roi = ee.FeatureCollection("FAO/GAUL/2015/level0")
+  .filter(ee.Filter.eq('ADM0_NAME', 'Sudan'));
+
+Map.centerObject(roi,5);
+
+var pointVisParams = {
+  color: 'FF0000' // Red color
+};
+
+// Add the CSV points to the map
+Map.addLayer(table, pointVisParams, 'Conflict Points');
+
+var Table = ee.FeatureCollection("projects/ee-ismith17/assets/2020-2024_SUDAN");
+
+
+// 5 km buffers //
+
+// Define the property names in the CSV file containing latitude and longitude
+var latitudeProperty = 'latitude';
+var longitudeProperty = 'longitude';
+
+// Define the buffer distance (5 km in meters)
+var bufferDistance = 5000;
+
+// Apply a buffer to each point
+var bufferedPoints = table.map(function(feature) {
+  return feature.buffer(bufferDistance);
+});
+
+var pointVisParamsB = {
+  color: 'FFA500' // Orange color
+};
+
+// Visualize the buffered points on the map
+Map.addLayer(bufferedPoints, pointVisParamsB, '5km Buffers');
+
+
+// 10 km buffers //
+
+// Define the property names in the CSV file containing latitude and longitude
+var latitudeProperty = 'latitude';
+var longitudeProperty = 'longitude';
```

Comment:
These two variables have been defined on line 23 and 24 so these lines don't actually do anything and can be removed